lib/node_modules/@stdlib/complex/float64/base/div/test/test.assign.js

```diff
@@ -0,0 +1,967 @@
+	var re1;
+	var im1;
+	var re2;
+	var im2;
+	var qre;
+	var qim;
+	var out;
+	var i;
+	var q;
+
+	re1 = tinyPositiveImaginaryComponents.re1;
+	im1 = tinyPositiveImaginaryComponents.im1;
+	re2 = tinyPositiveImaginaryComponents.re2;
+	im2 = tinyPositiveImaginaryComponents.im2;
+	qre = tinyPositiveImaginaryComponents.qre;
+	qim = tinyPositiveImaginaryComponents.qim;
+	out = new Float64Array( 2 );
+
+	for ( i = 0; i < re1.length; i++ ) {
+		q = cdiv( re1[ i ], im1[ i ], re2[ i ], im2[ i ], out, 1, 0 );
+
+		if ( q[ 0 ] === qre[ i ] ) {
+			t.strictEqual( q[ 0 ], qre[ i ], 'returns expected real component' );
+		} else {
+			delta = abs( q[ 0 ] - qre[ i ] );
+			tol = EPS * abs( qre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. real: '+q[ 0 ]+'. expected: '+qre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( q[ 1 ] === qim[ i ] ) {
+			t.strictEqual( q[ 1 ], qim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = abs( q[ 1 ] - qim[ i ] );
+			tol = EPS * abs( qim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. imag: '+q[ 1 ]+'. expected: '+qim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes a complex quotient (tiny positive real components)', function test( t ) {
+	var delta;
+	var tol;
+	var re1;
+	var im1;
+	var re2;
+	var im2;
+	var qre;
+	var out;
+	var qim;
+	var i;
+	var q;
+
+	re1 = tinyPositiveRealComponents.re1;
+	im1 = tinyPositiveRealComponents.im1;
+	re2 = tinyPositiveRealComponents.re2;
+	im2 = tinyPositiveRealComponents.im2;
+	qre = tinyPositiveRealComponents.qre;
+	qim = tinyPositiveRealComponents.qim;
+	out = new Float64Array( 2 );
+
+	for ( i = 0; i < re1.length; i++ ) {
+		q = cdiv( re1[ i ], im1[ i ], re2[ i ], im2[ i ], out, 1, 0 );
+
+		if ( q[ 0 ] === qre[ i ] ) {
+			t.strictEqual( q[ 0 ], qre[ i ], 'returns expected real component' );
+		} else {
+			delta = abs( q[ 0 ] - qre[ i ] );
+			tol = EPS * abs( qre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. real: '+q[ 0 ]+'. expected: '+qre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( q[ 1 ] === qim[ i ] ) {
+			t.strictEqual( q[ 1 ], qim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = abs( q[ 1 ] - qim[ i ] );
+			tol = EPS * abs( qim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re1[i]+' + '+im1[i]+'i. y: '+re2[i]+' + '+im2[i]+'i. imag: '+q[ 1 ]+'. expected: '+qim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function handles large and small numbers', function test( t ) {
+	var expected;
+	var out;
+	var v;
+
+	out = new Float64Array( 2 );
+	v = cdiv( 1.0e308, 5.0e307, 1.0, 0.5, out, 1, 0 );
+	expected = new Float64Array( [ 1.0e308, 0.0 ] );
+	t.strictEqual( isAlmostEqualFloat64Array( v, expected, EPS ), true, 'returns expected values' );
```

Comment:
@aayush0325 This is still wrong. `EPS` is not what should be passed into `isAlmostEqualFloat64Array`. Please read the docs. Ref: https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/assert/is-almost-equal-float64array